source/releasenotes/changes.rst

```diff
@@ -14,7 +14,425 @@
    under the License.
 
 
-Changes in |release| since 4.18.0.0
+Changes in |release| since 4.18.1.0
+===================================
+
+Apache CloudStack uses GitHub https://github.com/apache/cloudstack/milestone/29?closed=1
+to track its issues.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+
++-------------------------+----------+---------------+----------+------------------------------------------------------------+
+| Version                 | Github   | Type          | Priority | Description                                                |
```

Comment:
Nit - maybe in the generator script (py), we can remove the type and priority columns as they are empty;  for example, you can also use this https://github.com/shapeblue/generate_acs_rn (I had to create the fork as I didn't have push rights to the original one)

Reply:
done